functions/firestore/UserHelper.js

```diff
@@ -7,39 +7,61 @@ const {
   ADMIN_AUTH
 } = require("../utils/constants");
 
-module.exports.createUser = (body, uid) =>
+const calculateReview = (previousFeedbackScore, newReview) => {
+  const initScore = previousFeedbackScore > 0
+    ? previousFeedbackScore
+    : 10;
+  return (initScore + (Object.values(newReview).filter(fb => fb === true)).length * 3) / 2;
```

Comment:
=== true nem kell, siman fb => fb is mukodik

Reply:
azert kell, mert benne van a `userId `is az objectben es a string is truthy